### PR TITLE
Initial deployment

### PR DESCRIPTION
**Pull request by gbirke**
Created the deployment scripts and documentation. They'll be improved as needed.

This is for https://phabricator.wikimedia.org/T123049

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
+1 for what it's worth - I don't know Ansible at all

**Comment by addshore** [comment]
Oooooh, Ansible ;)

File: deployment/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Deployment
+
+This directory contains the deployment files for the fundraising frontend application itself.
+
+## Prerequisites
+The application requires the following infrastructure to be set up already on the server:
+
+- PHP 7
+- Composer
+- A web server with the correct document root and php configuration
```
Comment (addshore) [inline deployment/README.md]: PHP ;)

**Comment by JeroenDeDauw** [comment]
> Oooooh, Ansible ;)

It works like this right?

enders_game_cover

**Comment by addshore** [comment]
Indeed, I havn't tested this at all but it all looks sane! :)

**Comment by KaiNissen** [comment]
Life reviewed this five times already, merging now.

**Pull request merged**